src/java/org/apache/cassandra/db/compaction/Verifier.java

```diff
@@ -543,15 +545,17 @@ public LongPredicate getPurgeEvaluator(DecoratedKey key)
         public final boolean mutateRepairStatus;
         public final boolean checkOwnsTokens;
         public final boolean quick;
+        public final Function<String, ? extends Collection<Range<Token>>> tokenLookup;
```

Comment:
At the very least add a comment that documents the function. Ideally, you could just create a simple Interface/Class that can wrap this and have a meaningful method invocation like this: `Tokens.forKeyspace(String name)`

Reply:
I can add some docs, but think its best to use function rather than a new interface for this.

Reply:
added docs @dineshjoshi